backend/tauri/src/core/service/control.rs

```diff
@@ -11,7 +11,10 @@ pub async fn get_service_install_args<'n>() -> Result<Vec<OsString>, anyhow::Err
     let user = {
         #[cfg(windows)]
         {
-            nyanpasu_utils::os::get_current_user_sid().await?
```

Comment:
Hi. Please ensure the output eirther sid or username, not both.

Reply:
https://github.com/libnyanpasu/nyanpasu-utils/pull/25